scripts/bootstrap.sh.in

```diff
@@ -58,6 +58,7 @@ SALT_CALL=${SALT_CALL:-salt-call}
 declare -a PACKAGES=(
     salt-minion
     genisoimage
+    jq
```

Comment:
That mean solution script will not work if you upgrade from a previous version where `jq` were not installed during bootstrap (ok for me if we do the python stuff before next release)

Reply:
>  if we do the python stuff before next release

We should be able to, as it's going to be our next priority (#2163).

Reply:
done